semcore/breadcrumbs/src/style/breadcrumbs.shadow.css

```diff
@@ -3,46 +3,59 @@ SBreadcrumbs {
   align-items: center;
   padding: 0;
   margin: 0;
-  list-style: none;
-}
 
-SSeparator {
-  margin: 0 var(--intergalactic-spacing-2x, 8px);
-  color: var(--intergalactic-icon-secondary-neutral, #a9abb6);
-  display: flex;
+  SOl {
```

Comment:
Probably more semantic name is needed here

Reply:
changed